src/Tribe/Integrations/Manager.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * Class Tribe__Tickets__Integrations__Manager
+ *
+ * Loads and manages the third-party plugins integration implementations.
+ *
+ * @since TBD
+ */
+class Tribe__Tickets__Integrations__Manager {
+
+	/**
+	 * The current instance of the object.
+	 *
+	 * @since TBD
+	 *
+	 * @var Tribe__Tickets__Integrations__Manager
+	 */
+	protected static $instance;
+
+	/**
+	 * The class singleton constructor.
+	 *
+	 * @since TBD
+	 *
+	 * @return Tribe__Tickets__Integrations__Manager
+	 */
+	public static function instance() {
+		if ( empty( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Conditionally loads the classes needed to integrate with third-party plugins.
+	 *
+	 * Third-party plugin integration classes and methods will be loaded only if
+	 * supported plugins are activated.
+	 *
+	 * @since TBD
+	 */
+	public function load_integrations() {
+		$this->load_freemius();
+	}
+
+	/**
+	 * Loads our Freemius integration
+	 *
+	 * @since TBD
+	 */
+	private function load_freemius() {
+		tribe_singleton( 'tickets.integrations.freemius', new Tribe__Tickets__Integrations__Freemius );
+	}
```

Comment:
:no_entry_sign: Parenthesis should always be used when instantiating a new object.

<sub>hash: 6ef9d762381e6cd59bc149a36127cfed</sub>